src/Stripe.net/Properties/InternalsVisibleTo.cs

```diff
@@ -1,3 +1,4 @@
 ﻿using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("Stripe.net.Tests")]
+[assembly: InternalsVisibleTo("Stripe.net.Tests.xUnit")]
```

Comment:
i imagine this was an artifact that worked with AccountJSON. This is fine by me, just want to confirm it isn't needed anymore.